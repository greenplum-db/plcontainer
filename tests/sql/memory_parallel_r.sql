CREATE OR REPLACE FUNCTION r_memory_parallel(num int) RETURNS int2 AS $$
# container: plc_r_shared
a <- matrix(1, ncol=1024, nrow=1024*num)
return (1)
$$ LANGUAGE plcontainer;

CREATE TABLE NUM_OF_LOOPS_R (num int, aux int);
INSERT into NUM_OF_LOOPS_R (num, aux) select 10 as num, aux from generate_series(1, 128) as aux;
INSERT into NUM_OF_LOOPS_R (num, aux) select 9 as num, aux from generate_series(1, 128) as aux;
