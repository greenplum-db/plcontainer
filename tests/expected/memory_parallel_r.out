CREATE OR REPLACE FUNCTION r_memory_parallel(num int) RETURNS int2 AS $$
# container: plc_r_shared
a <- matrix(1, ncol=1024, nrow=1024*num)
return (1)
$$ LANGUAGE plcontainer;
CREATE TABLE NUM_OF_LOOPS_R (num int, aux int);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'num' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
INSERT into NUM_OF_LOOPS_R (num, aux) select 20 as num, aux from generate_series(1, 128) as aux;
INSERT into NUM_OF_LOOPS_R (num, aux) select 19 as num, aux from generate_series(1, 128) as aux;
